<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditStringCellDialog</class>
 <widget class="QDialog" name="EditStringCellDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QPushButton" name="saveButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Save changes &amp; close editor</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Close editor</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>77</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Undo all changes</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previousButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>62</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Previous string</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="qtbleditor.qrc">
     <normaloff>:/QTblEditor/Resources/Up.png</normaloff>:/QTblEditor/Resources/Up.png</iconset>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Up</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>62</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Next string</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="qtbleditor.qrc">
     <normaloff>:/QTblEditor/Resources/Down.png</normaloff>:/QTblEditor/Resources/Down.png</iconset>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Down</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="qtbleditor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>EditStringCellDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>EditStringCellDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
